### PR TITLE
docker: update to 20.10.16

### PR DESCRIPTION
**Pull request by tjhexf**

### CODE REVIEW

File: root-packages/docker/build.sh [hunk 1]
```diff
@@ -2,14 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://docker.com
 TERMUX_PKG_DESCRIPTION="Set of products that use OS-level virtualization to deliver software in packages called containers."
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=20.10.14
+TERMUX_PKG_VERSION=20.10.15
 LIBNETWORK_COMMIT=9db86fb7b8d62a276cb784bf9c4520bf8187dd19
 DOCKER_GITCOMMIT=a224086
```
Comment (xtkoba) [inline root-packages/docker/build.sh:7]: Tag `v20.10.15` seems to be associated with commit `fd82621d35d2d9662854c8351976d5cc1e4186ce`.

**Review by thunder-coding — APPROVED** [review]

**Comment by thunder-coding** [comment]
Seems good! If any root user can confirm whether the built debs from CI artifacts work fine, I'll merge this

**Comment by xtkoba** [comment]
I do not commit this solely because I do not have access to any environment to test this on. I would commit this without testing if there were published CVEs for the previous version. Sorry for your inconvenience.

**Comment by Grimler91** [comment]
Thanks! I don't have a device with a docker-compatible kernel either at the moment, but most likely a minor update should be fine

**Pull request merged**